src/main/kotlin/com/beust/klaxon/Render.kt

```diff
@@ -10,7 +10,17 @@ private fun <A: Appendable> A.renderString(s: String): A {
             '\\' -> append(ch).append(ch)
             '\n' -> append("\\n")
             '\r' -> append("\\r")
-            else -> append(ch)
+            else -> {
+                if(isPrintableUnicode(ch)){
```

Comment:
why do we escape `\b` and `\f` with unicode wscape? should't we escape them properly as for `\n` ?